clients/sail_ui/lib/rpcs/bitnames_rpc.dart

```diff
@@ -83,6 +83,96 @@ class BitnamesLive extends BitnamesRPC {
 
   @override
   Future<void> stopRPC() async {
-    await _client().call('stop');
+    try {
+      await _client().call('stop');
+      await Future.delayed(const Duration(milliseconds: 500));
+    } catch (e) {
+      if (e.toString().contains('Connection refused') || e.toString().contains('Unknown Error')) {
+        return;
+      }
+      log.e('Error stopping RPC', error: e);
+      rethrow;
+    }
+  }
```

Comment:
this should be simple and just call 'stop'. More complex logic is handled in base rpc_connection.dart